### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by gidden**
rm windows guide and add conda depedency install instructions

goes with cyclus/cyclus.github.com#156

### CODE REVIEW

**Comment by gidden** [comment]
ping (this was a travis internal error. only file changed is the readme)

**Comment by mbmcgarry** [comment]
Still failing on Travis?

**Comment by gidden** [comment]
The travis failure was an internal error. This only touches documentation.

On Wed, Jun 3, 2015 at 12:33 PM, mbmcgarry <notifications@github.com> wrote:

> Still failing on Travis?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1178#issuecomment-108538192>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

File: README.rst [hunk 1]
```diff
@@ -9,17 +9,29 @@ framework upon which to develop innovative fuel cycle simulations.
...
+If you want to get up and running as quickly as possible:
+
+#. Install `VirtualBox <https://www.virtualbox.org/>`_
+#. Download the `environment
+   <http://cnergdata.engr.wisc.edu/cyclus/virtual-box/current/cyclus-conda-devs.ova>`_
+#. Open the .ova file using VirtualBox (File->Import Appliance).
+#. Login with the username ``ubuntu`` and pw ``reverse``
+#. Skip to "Installing Cyclus"
+
+Note that the cyclus source code exists already on the virtual machine. You can
```
Comment (mbmcgarry) [inline README.rst]: cyclus->Cyclus

File: README.rst [hunk 2]
```diff
@@ -9,17 +9,29 @@ framework upon which to develop innovative fuel cycle simulations.
...
+update it with
+
+.. code-block:: console
+
+    $ cd cyclus
+	$ git pull origin cyclus
 
 ******************************
-Building and Installing Cyclus
+Depedencies
```
Comment (mbmcgarry) [inline README.rst]: Dependencies

File: README.rst [hunk 3]
```diff
@@ -62,9 +74,41 @@ tcmalloc (for speed)   any?
 *Note that the Debian/Ubuntu package `libtcmalloc` is NOT discovered correctly
 by our build system.  Instead use `libgoogle-perftools-dev`.*
 
+Installing Dependencies
+=========================
+
+Cyclus depedencies can either be installed via a operating system's pacakge
```
Comment (mbmcgarry) [inline README.rst]: dependencies

File: README.rst [hunk 4]
```diff
@@ -62,9 +74,41 @@ tcmalloc (for speed)   any?
...
+manager or via Conda.
+
+Via Conda
+---------
+
+*Note, at present, Conda dependency downloading is only supported for Linux*
+
+A conda installation may be the most straightforward for a new developer. These
+instructions will be similar to those shown for Cyclus user's `conda
```
Comment (mbmcgarry) [inline README.rst]: users (no apostrophe)

File: README.rst [hunk 5]
```diff
@@ -62,9 +74,41 @@ tcmalloc (for speed)   any?
...
+
+    .. code-block:: console
+
+        $ bash Miniconda-3.5.2-Linux-x86_64.sh -b -p ~/miniconda
+
+#. Add the following line to your ``.bashrc`` (``.bash_profile`` for Mac users):
+
+    ``export PATH-"${HOME}/miniconda/bin:${PATH}"``
+
+#. Install the depedencies
```
Comment (mbmcgarry) [inline README.rst]: dependencies

File: README.rst [hunk 6]
```diff
@@ -114,10 +158,10 @@ dependencies:
 
 .. code-block:: bash
 
-   sudo apt-get install -y cmake make libboost-all-dev libxml2-dev libxml++2.6-dev libsqlite3-dev libhdf5-serial-dev libbz2-dev coinor-libcbc-dev coinor-libcoinutils-dev coinor-libosi-dev coinor-libclp-dev coinor-libcgl-dev libblas-dev liblapack-dev g++ libgoogle-perftools-dev
+   sudo apt-get install -y cmake make libboost-all-dev libxml2-dev libxml++2.6-dev libsqlite3-dev libhdf5-serial-dev libbz2-dev coinor-libcbc-dev coinor-libcoinutils-dev coinor-libosi-dev coinor-libclp-dev coinor-libcgl-dev libblas-dev liblapack-dev g++ libgoogle-perftools-de
```
Comment (mbmcgarry) [inline README.rst]: should this end with 'de' or 'dev' ?

**Comment by mbmcgarry** [comment]
I get an error when I click on: 
2. Download the environment

"You don't have permission to access /cyclus/virtual-box/current/cyclus-conda-devs.ova on this server."
Intentional?

File: README.rst [hunk 7]
```diff
@@ -62,9 +74,41 @@ tcmalloc (for speed)   any?
...
+
+Cyclus depedencies can either be installed via a operating system's pacakge
+manager or via Conda.
+
+Via Conda
+---------
+
+*Note, at present, Conda dependency downloading is only supported for Linux*
+
+A conda installation may be the most straightforward for a new developer. These
```
Comment (mbmcgarry) [inline README.rst]: Conda

**Comment by mbmcgarry** [comment]
'Running Cyclus' has a broken link to `Cyclus Users Guide`_ (though this isn't part of your changes).  Same issue in Developer Workflow  `Updating Your Repositories`_

**Comment by mbmcgarry** [comment]
'Reviewing Pull Requests' link to style guidelines points to an old Cyclus page.  I think we've replaced this with a newer one?

**Comment by mbmcgarry** [comment]
Link under 'See Also' is broken as well

**Comment by mbmcgarry** [comment]
Hi Matt,
The comments above extend beyond the original scope of your PR.  I know it's extra work, but I think it's logical to include them all in this PR and important to have everything up-to-date for the workshops

**Comment by mbmcgarry** [comment]
Looks good, thanks @gidden !

**Pull request merged**